app/controllers/playlists_controller.rb

```diff
@@ -0,0 +1,81 @@
+class PlaylistsController < ApplicationController
+
+  def index
+
+  end
+
+  def new
+    @playlist = Playlist.new
+
+  end
+
+  def show
+    @playlist = Playlist.find(params[:id])
+    @client = Grooveshark::Client.new({session: session[:groove_session]})
+    @picture = @playlist.art_url
+    unless @picture
```

Comment:
This feels like the responsibility of the model - it should default the art_url if there isn't one provided.